Configuration/Generator/python/#OmegaMinus_13p6TeV_SoftQCDInel_TuneCUEP8M1_cfi.py#

```diff
@@ -0,0 +1,39 @@
+# Found 37 output events for 40000 input events.
+# Filter efficiency = 0.000925
+# Timing = 0.015818 sec/event
+# Event size = 456.1 kB/event
```

Comment:
This file shouldn't be committed, no?

Reply:
doh :( sorry, I did not realise it, I'm going to clean